src/NearOrg/FoundersPage.jsx

```diff
@@ -0,0 +1,277 @@
+  background-size: 75px 75px;
+  background-repeat: repeat;
+  background-position: center top;
+
+  @media (max-width: 900px) {
+    min-height: 390px;
+  }
+`;
+
+const PatternContent = styled.div`
+  padding: 1rem;
+  max-width: 496px;
+  margin: 0 auto;
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  min-height: 260px;
+
+  @media (max-width: 900px) {
+    min-height: 0px;
+  }
+`;
+
+const IconCircle = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
+  border-radius: 100%;
+  border: 1px solid var(--sand11);
+
+  i {
+    color: var(--sand11);
+    font-size: 32px;
+  }
+`;
+
+const Resource = ({ name, description, icon, url, target }) => (
+  <Widget
+    src="${REPL_ACCOUNT}/widget/NearOrg.Card"
+    key={name}
+    props={{
+      as: "a",
+      href: url,
+      target,
+      rel: "noopener noreferrer",
+      alt: name,
+      children: (
+        <>
+          <IconCircle>
+            <i className={`ph-duotone ${icon}`} />
+          </IconCircle>
+
+          <Flex direction="column" gap="16px">
+            <Text size="text-l" fontWeight="500">
+              {name}
+            </Text>
+            <Text size="text-s">{description}</Text>
+          </Flex>
+        </>
+      ),
+    }}
+  />
+);
+
+const selfServeResources = [
+  {
+    name: "Content Repository",
+    description:
+      "Here you will find the templates, guides, and additional resources essential for a founder's journey.",
+    icon: "ph-article",
+    url: "https://near-horizon.notion.site/NEAR-Founder-Hub-Content-Database-d69fcf11613443128fa249fe9fa069f0",
+    target: "_blank",
+  },
+  {
+    name: "Founder Co-Pilot",
+    description:
+      "Engage with interactive technical and business support to answer specific questions that you will likely encounter while building your product and company.",
+    icon: "ph-github-logo",
+    url: "https://near-cp-alpha.vercel.app",
+    target: "_blank",
+  },
+  {
+    name: "Token Launch Resources",
+    description: "Discover detailed support to help you understand how to launch a token.",
+    icon: "ph-file-doc",
+    url: "https://drive.google.com/file/d/1-UkMbSj1GGqO4aa9LfkrTwVvscLYZkJ7/view",
```

Comment:
@shelegdmitriy this link is broken. We should use the same pattern as we use for the PDFs at `/papers`